src/dynamicTypes/eventHandlersService.ts

```diff
@@ -3,7 +3,8 @@ import {
   ComponentsEventHandlers,
   EventHandler
 } from "../../scripts/generate-components-handlers/event-handlers-parser";
-
+// why do we need to import and export this interface
```

Comment:
i moved this to the eventHandlerService.
Makes more sense to me for it to be there